samples/crypto/kmu_usage_nrf54l/boards/nrf54l15dk_nrf54l05_cpuapp.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Using hardware crypto accelerator
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=n
+CONFIG_PSA_CRYPTO_DRIVER_CRACEN=y
+
+CONFIG_CRACEN_IKG=y
+
+CONFIG_HW_UNIQUE_KEY=y
+
+# Required to automatically generate and store SEED value during the first boot
+CONFIG_HW_UNIQUE_KEY_WRITE_ON_CRYPTO_INIT=y
```

Comment:
These are required for every device right? So might as well put them in the prj.conf. Since it is a sample we want to show the minimal configuration needed